src/coffee/popup.coffee

```diff
@@ -69,13 +123,14 @@ class Popup
     @clientsPanel = new ClientsPanel(@elem.find("#{prefix}-clients"), @)
     @finishPanel = new FinishPanel(@elem.find("#{prefix}-finish"), @)
 
-  moveClients: (amount) ->
-    @clientsPanel.moveLeft(amount)
-
-  movePodcast: (amount) ->
-    @podcastPanel.moveLeft(amount)
-
-  moveFinish: (amount) ->
-    @finishPanel.moveLeft(amount)
+  movePanels: (step) ->
+    @container = @elem.find('#podlove-subscribe-popup-modal-inner')
+    if @container.hasClass('swiped-left-0')
```

Comment:
I think the `if`s can be removed here. `removClass` won't complain if the class is not there.